backend/src/config/index.ts

```diff
@@ -4,7 +4,8 @@ import dotenv from 'dotenv'
 dotenv.config()
 
 const constants = {
-  DB_VERSION: '0019-replace_login_user_id_with_state_user_id',
+  DB_VERSION: '0020-delete_decay_start_block',
+  DECAY_START_TIME: new Date('2021-05-13 17:46:31'), // GMT+0
```

Comment:
I like this. But we need a test that ensures that this can never be changed on master.